packages/jellyfish-crypto/package.json

```diff
@@ -14,6 +14,7 @@
     "build": "tsc -b ./tsconfig.build.json"
   },
   "dependencies": {
+    "@defichain/jellyfish-transaction": "^0.0.0",
```

Comment:
This will cause cyclic dependency, you need to remove this.

Reply:
Thanks @fuxingloh! Moved the change inside `jellyfish-address` instead. LMK if this is fine 🙏